test/tools/utils.js

```diff
@@ -21,4 +21,5 @@ module.exports.sorted = (iterable, how) => {
   return items;
 };
 
-module.exports.oneMicroTask = promisify(queueMicrotask);
+const asyncMicroTask = promisify(queueMicrotask);
```

Comment:
I think exporting something with a different name is bad practice - why not just name this `runMicroTask`?

Reply:
Good call, actually not sure why I broke this across two lines the promisify will be done once on first load.